frontend/lib/location/determine_position.dart

```diff
@@ -106,32 +110,48 @@ Future<LocationStatus> checkAndRequestLocationPermission(
         await Geolocator.openLocationSettings();
       }
     }
-    
+
     if (!await Geolocator.isLocationServiceEnabled()) {
       return LocationStatus.notSupported;
     }
   }
 
   if (requestIfNotGranted) {
     var permission = await Geolocator.checkPermission();
-    
+
     if (permission == LocationPermission.deniedForever) {
-      // denied forever -> open UI
-      if (await openSettingsToGrantPermissions(context)) {
-        // get new result after the dialog is flo
-        var permission = await Geolocator.checkPermission();
-        return permission.toLocationStatus();
-      }
```

Comment:
Imo this is still the right thing to do. At least Google Maps does it the same (more or less)